src/ints/int10_memory.cpp

```diff
@@ -149,49 +150,41 @@ void INT10_LoadFont(const PhysPt _font, const bool reload, const int count,
 		real_writew(BIOSMEM_SEG, BIOSMEM_PAGE_SIZE, bios_pagesize);
 
 		// Move up one line on 14+ line fonts
-		auto cursor_height = height >= 14 ? (height - 1) : height;
+		auto cursor_height = char_height >= 14 ? (char_height - 1) : char_height;
 
 		INT10_SetCursorShape(cursor_height - 2, cursor_height - 1);
 	}
 }
 
 void INT10_ReloadFont()
 {
-	constexpr auto Reload   = false;
-	constexpr auto NumChars = 256;
-	constexpr auto Offset   = 0;
-	constexpr auto Map      = 0;
+	constexpr auto Reload    = false;
+	constexpr auto NumChars  = 256;
+	constexpr auto FirstChar = 0;
+	constexpr auto FontBlock = 0;
 
 	switch (CurMode->cheight) {
-	case 8:
+	case 8: {
+		const auto font_data = RealToPhysical(int10.rom.font_8_first);
+		constexpr auto CharHeight = 8;
 
-		INT10_LoadFont(RealToPhysical(int10.rom.font_8_first),
-		               Reload,
-		               NumChars,
-		               Offset,
-		               Map,
-		               CurMode->cheight);
-		break;
+		INT10_LoadFont(font_data, Reload, NumChars, FirstChar, FontBlock, CharHeight);
+	} break;
 
-	case 14:
-		INT10_LoadFont(RealToPhysical(int10.rom.font_14),
-		               Reload,
-		               NumChars,
-		               Offset,
-		               Map,
-		               CurMode->cheight);
-		break;
+	case 14: {
+		const auto font_data      = RealToPhysical(int10.rom.font_14);
+		constexpr auto CharHeight = 14;
 
-	case 16:
+		INT10_LoadFont(font_data, Reload, NumChars, FirstChar, FontBlock, CharHeight);
+	} break;
+
+	case 16: {
 	default:
-		constexpr auto Height = 16;
-		INT10_LoadFont(RealToPhysical(int10.rom.font_16),
-		               Reload,
-		               NumChars,
-		               Offset,
-		               Map,
-		               Height);
-		break;
+		const auto font_data      = RealToPhysical(int10.rom.font_16);
+		constexpr auto CharHeight = 16;
+
+		INT10_LoadFont(font_data, Reload, NumChars, FirstChar, FontBlock, CharHeight);
+	} break;
 	}
```

Comment:
No complaints here; but noticed the switch-case only really controlling one variable, so allows for something like this:

```c++
void INT10_ReloadFont()
{
	constexpr auto Reload    = false;
	constexpr auto NumChars  = 256;
	constexpr auto FirstChar = 0;
	constexpr auto FontBlock = 0;

	const auto char_height = CurMode->cheight;

	const auto font_data = RealToPhysical(
		char_height == 8  ? int10.rom.font_8_first :
		char_height == 14 ? int10.rom.font_14 :
		                    int10.rom.font_16
  );

	INT10_LoadFont(font_data, Reload, NumChars, FirstChar, FontBlock, char_height);
}
```

I find it a bit easier to read and quickly see the variables in-play, and the ternary worm reads like a switch/case.  

Edit - I see one bool added in the last commit, so might be doable. Just wanted to toss it as an option :-)

Reply:
That's a good idea @interloper98, I'll condense the switch/case block as suggested.

Reply:
Done.